homework/week-2/newStyle.css

```diff
@@ -0,0 +1,100 @@
+.header {
+  background-color: #176ab0;
+  color: white;
+  text-align: center;
+  padding: 5px;
+}
+
+#main-content {
+  width: 350px;
+  float: left;
+  padding: 10px;
+}
+
+#footer {
+  background-color: #176ab0;
+  color: white;
+  clear: both;
+  text-align: center;
+  padding: 5px;
+}
+
+.badge {
+  background: grey;
+  float: right;
+  color: white;
+  width: 18px;
+  height: 18px;
+  text-align: center;
+  line-height: 18px;
+  border-radius: 50%;
+  padding: 10px;
+}
+
+.flex-wrapper {
+  font-family: Abel;
+  background-color: grey;
+  text-align: left;
+  padding-top: 25px;
+  padding-bottom: 25px;
+  font-size: 20px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  background-color: #2E2C2F;
+  color: #444;
+}
+
+#contributorsDiv.li {
```

Comment:
This doesn't do anything; you're trying to style an element with the id `contributorsDiv` and the class `li`. It never exists in your app.